backend/src/main/java/gov/cdc/usds/simplereport/service/OrganizationInitializingService.java

```diff
@@ -387,4 +391,33 @@ private void createPatientSelfRegistrationLinkWithOrg(
       savePatientSelfRegistrationLink(p, prl);
     }
   }
+
+  public void createPatients(List<ConfigPatient> patients) {
+    if (patients != null && !patients.isEmpty()) {
+      for (ConfigPatient p : patients) {
+        Optional<Organization> orgLookup = _orgRepo.findByExternalId(p.getOrganizationExternalId());
+
+        if (!orgLookup.isPresent()) {
+          return;
+        }
+
+        Organization org = orgLookup.get();
+        String fullName = String.format("%s %s", p.getFirstName(), p.getLastName());
+        Optional<Person> foundPatient =
+            _personRepository.findAll().stream()
+                .filter(
+                    fp ->
+                        fp.getLastName().equals(p.getLastName())
+                            && fp.getFirstName().equals(p.getFirstName())
+                            && !fp.isDeleted())
```

Comment:
for consideration: do we want to recreate the patient profiles here if they're explicitly soft-deleted? Could imagine a situation where someone is testing something and have explicitly deleted all the patients and getting surprised that the database gets reseeded. 

Don't think that situation would be too common, but maybe we just recreate when the table explicitly is empty.

Reply:
actually did test this briefly and it looks like by default `findAll` just returns non-deleted entities, so this line isn't needed. Question still stands though :)

Reply:
ahhh thanks for checking! I'll remove the `!fp.isDeleted()` check then!!